documentation/projects/proposals/publish_dataset/20230719-implementation_plan_initial_data_dump.md

```diff
@@ -0,0 +1,411 @@
+# 2023-07-19 Implementation Plan: Initial Data Dump
+
+**Author**: @aetherunbound
+
+<!-- See the implementation plan guide for more information: https://github.com/WordPress/openverse/tree/19791f51c063d0979112f4b9f4eeace04c8cf5ff/docs/projects#implementation-plans-status-in-rfc -->
+<!-- This template is exhaustive and may include sections which aren't relevant to your project. Feel free to remove any sections which would not be useful to have. -->
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat
+- [ ] @stacimc
+- [ ] @apolinario (optional)
+- [ ] @Skylion007 (optional)
+- [ ] @zackkrida (optional)
+
+## Project links
+
+<!-- Enumerate any references to other documents/pages, including milestones and other plans -->
+
+- [Project Thread](https://github.com/WordPress/openverse/issues/2545)
+- [Project Proposal](https://docs.openverse.org/projects/proposals/publish_dataset/20230706-project_proposal.html)
+
+## Overview
+
+<!-- An overview of the implementation plan, if necessary. Save any specific steps for the section(s) below. -->
+
+This plan describes how the initial data dump of the media tables within
+Openverse's API database will occur on a technical level, as well as information
+regarding marketing and documentation of the initial data dump.
+[HuggingFace](https://huggingface.co/) has graciously offered to host the
+dataset for free, and this plan assumes that a handoff of our data to the
+HuggingFace team will be made to accomplish this.
+
+## Expected Outcomes
+
+<!-- List any succinct expected products from this implementation plan. -->
+
+Upon the completion of this implementation plan, the following should be
+available:
+
+- An unscheduled DAG which is capable of generating the data dump in S3.
+- A copy of the Openverse API database's primary media tables (i.e. `image` and
```

Comment:
We'd be happy to provide access to a Bucket to dump everything too if that is a blocker.